firefly/kernel/init/init.cpp

```diff
@@ -40,8 +41,7 @@ static stivale2_header_tag_framebuffer framebuffer_hdr_tag = {
 struct stivale2_header_tag_terminal terminal_hdr_tag = {
     .tag = {
         .identifier = STIVALE2_HEADER_TAG_TERMINAL_ID,
-        .next       = (uint64_t)&framebuffer_hdr_tag
```

Comment:
static / reinterpret cast

Reply:
Sure, I copied it from the example C code so that's why you  see that C style cast